nss-day-cohort-52/giffygram-wild-hurricanes
# Csl- save post state and also dynamically render the date posted on each saved post

#### Issues Completed
Applies to issue #26
Applies to issue #24 

#### Changes Made
1.  [X] Feature
    [] Bug-fix
    [] Feature that will cause a bug
    [] Not a feature or a bug- requires documentation
2. Added file `filename` to `directoryname` directory.
Did not create any new files 
3. Modified file `provider.js, postEntry.js, postList.js ` to save the Post state to the API and dynamically render the date posted on each saved post 
4. Make sure your code has been commented.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    commit your current code (git add --all, git commit -m "")
    git fetch --all
    git checkout csl-savePostState
    ```
2. Serve the code to browser.
3. Test app functionality.
    > How should the reviewer test functionality?
    > What is the intended outcome?
    > Try to create and save a post, it should show up at the bottom of the feed with today's date displayed next to the post author's name. Can double check in your database to make sure it is getting posted to the API correctly 
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
